internal/template.go

```diff
@@ -94,8 +93,16 @@ func maybeExecBackendsTpl(tplData *TplData, bcks Backends, genBcks bool) ([]byte
 		}
 	}
 
+	return deps
+}
+
+func maybeExecBackendsTpl(tplData *TplData, bcks Backends, genBcks bool) ([]byte, error) {
+	if !genBcks {
+		return nil, nil
+	}
+
 	clone := *tplData
```

Comment:
i see that this one already existed, not sure it's working this way, confusing to have both the tplData and clone in scope